examples/step-74/step-74.cc

```diff
@@ -262,16 +262,18 @@ namespace Step74
     void   compute_error_estimate();
     double compute_energy_norm();
 
-    Triangulation<dim>   triangulation;
-    const unsigned       degree;
-    QGauss<dim>          quadrature;
-    QGauss<dim - 1>      face_quadrature;
-    const MappingQ1<dim> mapping;
+    Triangulation<dim>    triangulation;
+    const unsigned        degree;
+    const QGauss<dim>     quadrature;
+    const QGauss<dim - 1> face_quadrature;
+    const QGauss<dim>     quadrature_2;
```

Comment:
I know why you want to do this, but I don't like the naming.

Reply:
I have renamed it to `quadrature_overintegration`. Feel free to suggest other names (it is used in the context to compute the error and the energy).